cg/cli/backup.py

```diff
@@ -34,6 +40,42 @@ def backup(context: CGConfig):
     pass
 
 
+@backup.command("flow-cells")
+@DRY_RUN
+@click.pass_obj
+def backup_flow_cells(context: CGConfig, dry_run: bool):
+    """Backup flow cells."""
+    pdc_api = PdcAPI(binary_path=context.pdc.binary_path, dry_run=dry_run)
+    status_db: Store = context.status_db
+    for flow_cell in get_flow_cells_from_path(flow_cells_dir=Path(context.flow_cells_dir)):
+        db_flow_cell: Optional[Flowcell] = status_db.get_flow_cell_by_name(
+            flow_cell_name=flow_cell.id
+        )
+        flow_cell_encryption_api = FlowCellEncryptionAPI(
+            binary_path=context.encryption.binary_path,
+            dry_run=dry_run,
+            encryption_dir=Path(context.backup.encryption_directories.current),
+            flow_cell=flow_cell,
+            pigz_binary_path=context.pigz.binary_path,
+            slurm_api=SlurmAPI(),
+            sbatch_parameter=context.backup.slurm_flow_cell_encryption.dict(),
+            tar_api=TarAPI(binary_path=context.tar.binary_path, dry_run=dry_run),
+        )
+        try:
+            pdc_api.start_flow_cell_backup(
+                db_flow_cell=db_flow_cell,
+                flow_cell_encryption_api=flow_cell_encryption_api,
+                status_db=status_db,
+            )
+        except (
+            DcmsAlreadyRunningError,
+            FlowCellAlreadyBackeupError,
+            FlowCellEncryptionError,
+            PdcError,
+        ) as error:
+            logging.debug(f"{error}")
```

Comment:
I agree that `DcmsAlreadyRunningError` and `FlowCellAlreadyBackeupError` can be logged with debug. But `FlowCellEncryptionError` and `PdcError` would be nice to have in the actual logs generated by the automation